exporter/azure_monitor/example/client.go

```diff
@@ -0,0 +1,42 @@
+package main
+// Package: Runs code for using Azure exporter
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"go.opencensus.io/exporter/azure_monitor"
+	"go.opencensus.io/exporter/azure_monitor/common"
+	"go.opencensus.io/trace"
+)
+
+func main() {
+	ctx := context.Background()
+
+	exporter, err := azure_monitor.NewAzureTraceExporter(common.Options{
+		InstrumentationKey: "11111111-1111-1111-1111-111111111111", // add your InstrumentationKey
+		EndPoint: 			"https://dc.services.visualstudio.com/v2/track",

```

Comment:
It will be great if this could be optional. By default we want people to write less code and get a reasonable outcome.

Reply:
I updated it so now you just need to enter a string for the iKey.